mmpose/datasets/pipelines/shared_transform.py

```diff
@@ -142,3 +149,132 @@ def __repr__(self):
         """Compute the string representation."""
         return (f'{self.__class__.__name__}('
                 f'keys={self.keys}, meta_keys={self.meta_keys})')
+
+
+@PIPELINES.register_module()
+class Albu(object):
+    """Albumentation augmentation (pixel-level transforms only) Adds custom
```

Comment:
Add period before `Adds`, and remove comma before `visit`

Reply:
OK.